__tests__/cli.test.js

```diff
@@ -117,3 +121,306 @@ describe.each(browsers)('Basic Test: %s', browser => {
+    expect(config).toBeTruthy();
+  });
+
+  it('Block one and two', async () => {
+    expect(config.options.block).toEqual(["one", "two"]);
+  });
+});
+
+describe('Two JSON block options', () => {
+  beforeAll(() => {
+    testId = null;
+    outputLogs = null;
+    config = null;
+
+    const safeBrowser = goodBrowser.replace(/[^a-z0-9-]/, '');
+
+    const args = [
+      'node',
+      'cli.js',
+      '--block',
+      '[ "one" ]',
+      '--block',
+      '[ "two" ]',
+      '--url',
+      'https://www.example.com',
+      '-b',
+      safeBrowser,
+    ];
+
+    const output = spawnSync(args[0], args.slice(1));
+    outputLogs = output.stdout.toString();
+    const match = outputLogs.match(/Test ID:(.*)/);
+    if (match && match.length > 1) {
+      testId = match[1].trim();
+    }
+    config = retrieveConfig(testId, safeBrowser);
+  });
+
+  it('generates a Configuration file', async () => {
+    expect(config).toBeTruthy();
+  });
+
+  it('Block one and two', async () => {
+    expect(config.options.block).toEqual(["one", "two"]);
+  });
+});
+
+describe('Two by 2 JSON block options', () => {
+  beforeAll(() => {
+    testId = null;
+    outputLogs = null;
+    config = null;
+
+    const safeBrowser = goodBrowser.replace(/[^a-z0-9-]/, '');
+
+    const args = [
+      'node',
+      'cli.js',
+      '--block',
+      '[ "one", "two" ]',
+      '--block',
+      '[ "three", "four" ]',
+      '--url',
+      'https://www.example.com',
+      '-b',
+      safeBrowser,
+    ];
+
+    const output = spawnSync(args[0], args.slice(1));
+    outputLogs = output.stdout.toString();
+    const match = outputLogs.match(/Test ID:(.*)/);
+    if (match && match.length > 1) {
+      testId = match[1].trim();
+    }
+    config = retrieveConfig(testId, safeBrowser);
+  });
+
+  it('Generates a Configuration file', async () => {
+    expect(config).toBeTruthy();
+  });
+
+  it('Block one, two, three and four', async () => {
+    expect(config.options.block).toEqual(["one", "two", "three", "four"]);
+  });
+});
+
+describe('Bad JSON block options', () => {
+  beforeAll(() => {
+    testId = null;
+    outputLogs = null;
+    config = null;
+
+    const safeBrowser = goodBrowser.replace(/[^a-z0-9-]/, '');
+
+    const args = [
+      'node',
+      'cli.js',
+      '--block',
+      "[ 'one', 'two' ]",
+      '--url',
+      'https://www.example.com',
+      '-b',
+      safeBrowser,
+    ];
+
+    const output = spawnSync(args[0], args.slice(1));
+    outputLogs = output.stdout.toString();
+    const errLogs = output.stderr.toString();
+    const match = outputLogs.match(/Test ID:(.*)/);
+    if (match && match.length > 1) {
+      testId = match[1].trim();
+    }
+    config = retrieveConfig(testId, safeBrowser);
+  });
+
+  it('generates a Configuration file', async () => {
+    expect(config).toBeTruthy();
+  });
+
+  it('Do not block one and two', async () => {
+    expect(config.options.block).toEqual([ ]);
```

Comment:
This should probably throw an input error instead of just ignoring the parameter.